Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemsController.cs

```diff
@@ -886,12 +888,12 @@ private ViewModelType PrepareProblemViewModelForEdit(int id)
                 .FirstOrDefault();
 
             var contest = problemEntity.FirstOrDefault()?.ProblemGroup.Contest;
-
             if (problem == null || contest == null)
             {
                 return null;
             }
-
+            
+            problem.DefaultSubmissionTypeId = problemEntity.FirstOrDefault().DefaultSubmissionTypeId;
```

Comment:
A couple lines above this I see the same  `problemEntity.FirstOrDefault()` call.

Reply:
fixed